ledger-core/v2/cmd/pulsard/main.go

```diff
@@ -104,6 +106,18 @@ func main() {
 	}
 
 	cm, server := initPulsar(ctx, pCfg)
+
+	if params.oneShot {
+		nextPulseNumber := pulse.OfNow()
+		err := server.Send(ctx, nextPulseNumber)
+		if err != nil {
+			panic(err)
+		}
+		// it's required since pulse is sent in goroutine
+		time.Sleep(time.Second * 10)
+		return
```

Comment:
I see `err = cm.Stop(ctx)` in defer. Do we need to stop component manager in one-shot run?

Reply:
There is no any action we can do if it's not stopped.
And it's the end of the program.
And code is shorter.
That is why I didn't call Stop().
But, for perfectness - yes.